### PR TITLE
Updating DOP value range for GPS notification

### PR DESCRIPTION
**Pull request by atiwari9**
This proposal is to update the range and make these fields non mandatory for DOP values in GPS notification for GetVehicleData and OnVehicleData RPCs.

### CODE REVIEW

**Comment by atiwari9** [comment]
@theresalech - This proposal is ready to be reviewed.

File: proposals/NNNN-Updating-DOP-value-range-for-GPS-notification.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    	<description>VDOP.  If value is unknown, value shall be set to 0.</description>
+	<history>
+    		<param name="vdop" type="Float" minvalue="0" maxvalue="65535" defvalue="0" mandatory="false" until="X.Y">
+    		</param>
+	</history>
+    </param>
+    <param name="actual" type="Boolean" mandatory="false">
+    	<description>
+    		True, if actual.
+    		False, if infered.
```
Comment (theresalech) [inline proposals/NNNN-Updating-DOP-value-range-for-GPS-notification.md]: Should this be "inferred" ?

File: proposals/NNNN-Updating-DOP-value-range-for-GPS-notification.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+None
+
+## Impact on existing code
+
+* RPC needs to be updated with new parameter
+* SDL core and mobile proxy lib need updates
+
+## Alternatives considered
+
+* Instead of increasing the maxvalue range, the maxvalue can be removed all together
```
Comment (theresalech) [inline proposals/NNNN-Updating-DOP-value-range-for-GPS-notification.md]: Please update to "altogether"

**Comment by atiwari9** [comment]
@theresalech  - Done

**Pull request merged**